tests/test_evolutionary_best_response.py

```diff
@@ -232,9 +232,7 @@ def test_get_evolutionary_best_response_with_K_equal_10_against_2_defectors():
     With K = 10 and 2 defectors the best response changes from `[1, 0.45420466,
     0, 0.03799109]` (for K = 1) to:
 
-    [0.26874769, 0.11697721, 0, 0]
-
-    TODO: Nik does this make sense?
+    [1, 0.36819048, 1, 0.75011483]
```

Comment:
This is better right? This makes more sense?

Reply:
Yes it does!

Reply:
🎉